docs/erc1155-skills.md

```diff
@@ -178,53 +184,34 @@ You should get `1000000000000000000000`.
 
 ## Update SOEF configurations for both AEAs
 
-To update the SOEF configuration, run in each AEA project:
+Update the SOEF configuration in both AEA projects:
 ``` bash
 aea config set vendor.fetchai.connections.soef.config.chain_identifier ethereum
 ```
 
 ## Run the AEAs
 
-First, run the deployer AEA.
+First, run the deployer AEA:
 
 ``` bash 
 aea run
 ```
 
-Once you see a message of the form `To join its network use multiaddr 'SOME_ADDRESS'` take note of the address.
+Once you see a message of the form `To join its network use multiaddr 'SOME_ADDRESS'` take note of the address. (Alternatively, use `aea get-multiaddress fetchai -c -i fetchai/p2p_libp2p:0.16.0 -u public_uri` to retrieve the address.)
+This is the entry peer address for the local <a href="../acn">agent communication network</a> created by the deployer.
 
-It will perform the following steps:
-- deploy the smart contract
-- create a batch of items in the smart contract
-- mint a batch of items in the smart contract
+This AEA then performs the following steps:
+
+ * deploys the smart contract
+ * creates a batch of items in the smart contract
+ * mints a batch of items in the smart contract
 
 At some point you should see the log output:
 ``` bash
 registering service on SOEF.
 ```
 
-Then, update the configuration of the client AEA's P2P connection by appending the following
-YAML text at the end of the `aea-config.yaml` file:
-
-``` yaml
----
-public_id: fetchai/p2p_libp2p:0.16.0
-type: connection
-config:
-  delegate_uri: 127.0.0.1:11001
-  entry_peers:
-  - SOME_ADDRESS
-  local_uri: 127.0.0.1:9001
-  log_file: libp2p_node.log
-  public_uri: 127.0.0.1:9001
-```
-
-where `SOME_ADDRESS` is the output
-of `aea get-multiaddress fetchai -c -i fetchai/p2p_libp2p:0.16.0 -u public_uri)` in the `erc1155_deployer` project.
-The output will be something like `/dns4/127.0.0.1/tcp/9000/p2p/16Uiu2HAm2JPsUX1Su59YVDXJQizYkNSe8JCusqRpLeeTbvY76fE5`.
```

Comment:
This sentence could be added above and kept, its useful for the user to have an example.